src/vumi2/workers.py

```diff
@@ -1,7 +1,7 @@
+import importlib.metadata
```

Comment:
This is only available from py3.8, but py3.7 is end-of-life in a couple of months anyway.